### ISSUE TITLE
backend error: GotNotify: Version mismatch! frontend="1.20.4" backend="1.20.2"

### ISSUE THREAD

**Issue by brijananand** [tailscale/tailscale#3927]
### What is the issue?

I'm unable to start tailscale up after upgrading from version 1.20.2-1 to 1.20.4-1. I use the community package available from [Arch repo](https://archlinux.org/packages/community/x86_64/tailscale/)

### Steps to reproduce

Command executed: 
```
yay -S tailscale
ackage (1)          Old Version  New Version  Net Change

community/tailscale  1.20.2-1     1.20.4-1       0.01 MiB

Total Installed Size:  27.65 MiB
Net Upgrade Size:       0.01 MiB

:: Proceed with installation? [Y/n] y
(1/1) checking keys in keyring                               [--------------------------------] 100%
(1/1) checking package integrity                             [--------------------------------] 100%
(1/1) loading package files                                  [--------------------------------] 100%
(1/1) checking for file conflicts                            [--------------------------------] 100%
:: Processing package changes...
(1/1) upgrading tailscale                                    [--------------------------------] 100%
:: Running post-transaction hooks...
(1/3) Reloading system manager configuration...
(2/3) Arming ConditionNeedsUpdate...
(3/3) Checking which packages need to be rebuilt
```

Error and warnings:
```
tailscale up
Warning: client version "1.20.4" != tailscaled server version "1.20.2"
2022/02/13 01:58:21 GotNotify: Version mismatch! frontend="1.20.4" backend="1.20.2"
backend error: GotNotify: Version mismatch! frontend="1.20.4" backend="1.20.2"
```

Checking Tailscale status:
```
tailscale status
Warning: client version "1.20.4" != tailscaled server version "1.20.2"
Tailscale is stopped.
```

### Are there any recent changes that introduced the issue?

Upgraded from version 1.20.2-1 to 1.20.4-1 on Endeavour OS (Arch Linux) running up-to-date system.

### OS

Linux

### OS version

EndeavourOS Linux x86_64 (Arch Linux) Kernel: 5.16.8-arch1-1

### Tailscale version

1.20.4

### Bug report

BUG-964d9a9910a62d6ac84e6ee42f9d9ab3bfd7530a64079b076b868b52e520fe7e-20220213071101Z-8750e72599fdc073

**Comment by DentonGentry** [comment]
I think the error is clear: the tailscale CLI is version 1.20.4 but the tailscaled server is 1.20.2.
The CLI is usually /usr/bin/tailscale, the server is usually /usr/sbin/tailscaled.

If something has gone wrong in installing the package, perhaps removing and re-adding or manually removing /usr/bin/tailscale and /usr/sbin/tailscaled would help correct it.

**Comment by bradfitz** [comment]
Or just restart the daemon. Usually the packaging does that.

**Comment by brijananand** [comment]
Thank you for your response @DentonGentry and @bradfitz.

I uninstalled tailscale, and did a search to find /usr/bin/tailscale and /usr/sbin/tailscaled but they did not exist, but every subsequent install still displayed the same error.

However, restarting the daemon did help. I also found that within /var/run/ there was the path to tailscaled's unix socket (default /var/run/tailscale/tailscaled.sock). I deleted that as well after the uninstallation, killed tailscaled pid, and reinstalled tailscale and now it works fine.

Thank you for your assistance!
